### PR TITLE
508 Compliance

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds an admonition to the index page for section 508/accessibility compliance.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12066

### CODE REVIEW

**Comment by Loquacity** [comment]
@jcayouette @juliogonzalez I just realised we need this in for 4.1.1, so let's get this merged and backported ASAP.

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -15,6 +15,23 @@ The Salt-master orchestrates tens of thousands of Salt clients ({productname} Cl
...
+If you cannot use the {productname} {webui}, almost all functionality (with the exception of virtualization tasks) is available from the command prompt.
+All command line tools can be used with assistive technology tools.
+Text readers ``orca`` and ``espeak`` can be enabled in the Desktop Applications Module.
+Speech recognition ``mycroft-core`` and ``sphinx`` are available in PackageHub.
+Additionally, you can interact with {productname} using the XML-RPC API.
+
+* For more information about [command]``spacecmd``, see xref:reference:spacecmd-intro.adoc[].
+* For more information about other command prompt tools, see xref:reference:command-line-tools.adoc[].
+* For more information about the API, see https://documentation.suse.com/external-tree/en-us/suma/4.1/pdf/susemanager_api_doc_color_en.pdf.
+* For the latest information about all command prompt tools, see the release notes available from https://www.suse.com/releasenotes/.
```
Comment (keichwa) [inline modules/ROOT/pages/index.adoc:30]:
```suggestion
* For the latest information about all command line tools, see the release notes available from https://www.suse.com/releasenotes/.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -15,6 +15,23 @@ The Salt-master orchestrates tens of thousands of Salt clients ({productname} Cl
...
+.Accessibility
+====
+If you cannot use the {productname} {webui}, almost all functionality (with the exception of virtualization tasks) is available from the command prompt.
+All command line tools can be used with assistive technology tools.
+Text readers ``orca`` and ``espeak`` can be enabled in the Desktop Applications Module.
+Speech recognition ``mycroft-core`` and ``sphinx`` are available in PackageHub.
+Additionally, you can interact with {productname} using the XML-RPC API.
+
+* For more information about [command]``spacecmd``, see xref:reference:spacecmd-intro.adoc[].
+* For more information about other command prompt tools, see xref:reference:command-line-tools.adoc[].
```
Comment (keichwa) [inline modules/ROOT/pages/index.adoc:28]:
```suggestion
* For more information about other command line tools, see xref:reference:command-line-tools.adoc[].
```
That's how you call them l.22.

**Comment by juliogonzalez** [comment]
> @jcayouette @juliogonzalez I just realised we need this in for 4.1.1, so let's get this merged and backported ASAP.

Go for it.

As for packaging, let's wait for @witekest first

**Comment by Loquacity** [comment]
:heavy_check_mark: backported to `manager-4.1` 48f86abcbf79921dfad1ebcebd6ed8520b16bb5d

**Pull request merged**